doc/design.md

```diff
@@ -273,6 +293,121 @@ Returns:
     * tbd
 
 
+### **`POST`** `/oeci_login` ?
+
+## Question
+
+What's the workflow for accessing OECI?
+
+Is it a webpage redirect, like the frontend prototype?
+
+Or is it an authentication serverside, like the current Crawler implementation?
+These behaviors need to be reconciled.
+
+I see three design versions:
+
+- Rely on a frontend page redirect, which obtains login access from the OECI directly to allow subsequent requests by Crawler. This requires a Crawler redesign, which currently passes creds to the remote site. It also requires caching a login token provided by OECI. Does the OECI login allow that, e.g. use a cookie?
+
+- Roll the login  into the `/search/` endpoint.
```

Comment:
I believe we had settled upon this version, and we could set a cookie on the client side that contains encrypted credentials that will get sent with each request. The cookie should expire after a very short time period.